frontend/online-shop/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./App.css";
+import ProductList from "./components/ProductList";
+import ProductDetailedView from "./components/ProductDetailedView";
+import { CssBaseline, ThemeProvider } from "@material-ui/core";
+import StandardTheme from "./themes/StandardTheme";
+import { BrowserRouter, Redirect, Route, Switch } from "react-router-dom";
+import Cart from "./components/Cart";
+// import ProductItem from "./components/ProductItem";
+
+function App() {
+  return (
+    <ThemeProvider theme={StandardTheme}>
```

Comment:
You should consider renaming your theme because it's no longer a standard one. `CustomTheme` would be more appropriate.